fronts-client/src/actions/Config.ts

```diff
@@ -2,7 +2,7 @@ import { Config } from 'types/Config';
 
 const configReceived = (config: Config) => ({
   type: 'CONFIG_RECEIVED',
-  payload: config
+  payload: config,
```

Comment:
I've seen automated code formatting apply this change a few times and not really understood it. What is the benefit of ending with a comma here? Is there a particular lint/prettier rule that enforces this?

Reply:
AFAIK Prettier has very few rules by design – instead it has opinions, and says, 'my way || highway' via reparsing the code, to [circumvent conversations about styling.](https://jlongster.com/A-Prettier-Formatter) (see 'Frictionless teams')

That said, I'm not sure why it has this particular opinion 😁